### DISCUSSION TITLE
Linux V6.0 test build

### DISCUSSION THREAD

**Discussion by gusmanb** [gusmanb/logicanalyzer#144] [General]
For Linux users that would like to test the new V6.0 and be part of the debugging, [here is the current build](https://mega.nz/file/yWxzzbqY#KiSTdKGng3rp4ou514D_QDEvaFoX18GSSkAnLFZtz9Y).

Remember to use one of the [new firmwares](https://mega.nz/file/3SZCDQzD#krl3m4G7HKwNuCi6rsmw_yeyi9CK_usz6gGNyXah-uY).

Python 3.x must be installed on the system, minimum version 3.7. If the analyzer is not able to locate the version you can add a "python.cfg" file in the app folder containing the path to the "libpython3.x.so".

If you find bugs or errors please notify them here.

## Keep in mind!

This is a test build and to ease things I will upload multiple versions with updates answering to the posts with bugs, please read the thread to find which version is the latest one.

Cheers.

**Comment by NobodyYouKnow** [comment]
By "here" I presume you mean in this discussion thread.

Color Picker anomalous reactivation in LogicAnalyzer v6.0 (repost)

In LogicAnalyzer v6.0, once Capture Settings -> Color Picker is activated and then closed (or Canceled), any subsequent click in the Capture Settings window will re-activate the Color Picker. This occurs in the Linux beta distribution. Issue can be cleared by closing the Capture Settings window.

**Reply by gusmanb** [reply]
Thanks.

Well, it seems that Avalonia on Linux doesn't likes what I'm doing, when I open a dialog from the PointerEvent it stucks the event and any other click event raises again the same one...

I have found a workaround but I don't like it as it is based in a delay and I think this may fail in a slower machine... in any case here is the version with the workaround (I also have changed the color picker itself to something more compact).

[Latest build](https://mega.nz/file/yWohnCIT#Jf3s3cyyteXjdXYgVAi7wehZ7Ju-iW__dEaUAozLU-8)

**Comment by tingox** [comment]
Are there any working Linux test builds of the software available now? Links posted in this thread are dead.

**Reply by gusmanb** [reply]
Wow, mega decided to invalidate the links... 😞 

Here you have all versions: https://mega.nz/folder/SGxDHAZL#afLGgQbJAaqOYXjhJBwokQ

**Reply by tingox** [reply]
That works - thank you.

**Comment by NobodyYouKnow** [comment]
Hmmm. That link leads me to a Mega folder titled "Empty Folder" containing a zero byte file named "Untitled.iso", all dated 2015. I am obviously missing something.

**Reply by gusmanb** [reply]
If you have clicked the link from the email notification it may be wrong.

**Reply by NobodyYouKnow** [reply]
The link posted above worked like a champ. Thank you!
